tests/charts/api_tests.py

```diff
@@ -1383,3 +1386,44 @@ def test_import_chart_invalid(self):
         assert response == {
             "message": {"metadata.yaml": {"type": ["Must be equal to Slice."]}}
         }
+
+    @pytest.mark.usefixtures("create_annotation_layers")
```

Comment:
We should think about moving all these tests away from `unittest.TestCase` if possible, there are some really cool pytest features that we are missing out

Reply:
+1, some functions like login, get_table_by_name should be already available as direct ( not object ) methods